docs/kintoblocks.md

```diff
@@ -43,6 +43,29 @@ The steps to tag a commit are:
 
 The main reason to tag a KintoBlock is to add it to an application, think of it as a release, so when you have your KintoBlock finished and want to update it in an application you can tag it and select the tag in the application. This is a way to control the released versions of your KintoBlock.
 
-## Custom Parameters (Coming Soon)
+## Custom Parameters
+
+Custom Parameters are another way to pass information to your **KintoApp**. These values are editable by the end user in the case of shared KintoBlocks, and also editable at a KintoApp level for cases where multiple apps are using the same **KintoBlock**.
```

Comment:
To your kintoblock microservice***

Reply:
Shared kintoblocks should be linked to thinking kinto: http://docs.kintohub.com/docs/thinking-kinto

Reply:
editable == overrideable per environment*

I think custom params need a clear intro

Custom params are used to do X.
You can override and modify the values per kintoapp environment, allowing you to have different values for development and production for example.
Due to shared kintoblocks, each API call you receive has context of who and what is calling you.  Configuration parameters are part of that context by adding custom key/value data to the headers of the API call.